### PR TITLE
Add README.md file

### PR DESCRIPTION
**Pull request by yhuan47**
First draft of README file, more details to be added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+What is DAI/DS?
+====================
+
+Thank you for your interest in DAI/DS!
+
+DAI/DS(Data Access Interface and Data Store) is a data-centric framework supporting a suite of domain-specific adapters that organizes data gathered from components. The data store is a multi-tier "container" that hosts data defined by UCS(Unified Control System) and made available to components.
```
Comment (paul-amonson) [inline README.md]: Space after UCS and before "(".

Comment (paul-amonson) [inline README.md]: Same for DAI/DS then the "("

Comment (yhuan47) [reply README.md]: Agree, will fix

Comment (yhuan47) [reply README.md]: Agree, will fix

**Comment by paul-amonson** [comment]
Looks good!

**Pull request merged**